src/cmd/ksh93/meson.build

```diff
@@ -99,6 +99,12 @@ if cc.has_function('sigqueue', prefix: '#include <signal.h>')
     shcomp_c_args += [ '-D_lib_sigqueue=1' ]
 endif
 
+if cc.has_function('backtrace', prefix: '#include <execinfo.h>',
```

Comment:
I'm going to reject this part of the PR for two reasons. First, it should be sufficient and is more efficient to just check if that header is available. Second, all but two of the `#ifdef _lib_backtrace` directives can be eliminated. If the interface isn't available it's simpler and safer to just define a dummy `dump_backtrace()` function that does nothing. Doing it that way means that if anyone ever adds an explicit call to that function (as opposed to the SIGSEGV code path) the code will still compile and run. But thanks for pointing out that we shouldn't just assume this feature is available.